render/tests/test-event.js

```diff
@@ -884,4 +916,48 @@ o.spec("event", function() {
 			})
 		})
 	})
+	o("avoid sync redraw after removal", function() {
```

Comment:
Can you extend this test to also verify the event listener itself wasn't called?

That way, we know it couldn't possibly attempt an async redraw later.

Reply:
I added assertions about whether the event listener itself was called or not.

The event listener itself is not removed and will be called, but it should be clearer that redraw is not being called.